src/main/scala/org/constellation/domain/trust/TrustData.scala

```diff
@@ -0,0 +1,5 @@
+package org.constellation.domain.trust
```

Comment:
@wkoszycki and I chatted briefly about this and I was thinking that since the SelfAvoidingWalk is probabilistic, perhaps we can call it trust. And validator rewards, which is consistent across nodes, reputation.